src/evaluation.py

```diff
@@ -352,10 +354,41 @@ def match_answer_for_question(self, current_score, question, marked_answer):
         )
         return delta
 
-    def conditionally_print_explanation(self):
+    def conditionally_print_explanation(self, file_id):
         if self.should_explain_scoring:
             console.print(self.explanation_table, justify="center")
 
+        self.explanation_to_csv(file_id)
+        self.explanation_table_data_for_csv = []
```

Comment:
Instead can we directory call a function `conditionally_save_explanation_csv()` from the parent? Wherever conditionally_print_explanation() is called currently?

Reply:
So, won't we show that explanation table as an output in the terminal?